libmenu/matemenu-tree.c

```diff
@@ -2056,7 +2056,7 @@ static gboolean load_parent_merge_file(MateMenuTree* tree, GHashTable* loaded_me
 	basedir   = menu_layout_node_root_get_basedir(root);
 	menu_name = menu_layout_node_root_get_name(root);
 
-	canonical_basedir = menu_canonicalize_file_name(basedir, FALSE);
+	canonical_basedir = realpath(basedir, NULL);
```

Comment:
Please add a space after `realpath` here, thank you.

Reply:
Will do (note that I just changed the function).